herchel/siervo.py

```diff
@@ -0,0 +1,122 @@
+class GradeManagementSystem:
+    def __init__(self):
+        self.students = {}  
+        self.subjects = [] 
+        self.max_grade = 100 
```

Comment:
capitalize constant name - max_grade

Reply:
change into capital letters for constant, fix in commit ac2f4b9 .